.github/workflows/cuda_reusable.yml

```diff
@@ -61,6 +66,19 @@ jobs:
           ${{ inputs.extra_build_args }} \
           -f docker/dev/Dockerfile .
 
+    - name: Build And Test TF Blade
+      run: |
```

Comment:
maybe we can use github workflow IF statement, that workflow do not need to schedule the job , ref: https://docs.github.com/en/actions/learn-github-actions/expressions#example-expression-in-an-if-conditional

Reply:
Done